nservicebus/scalability-and-ha/deploying-to-a-windows-failover-cluster.md

```diff
@@ -137,6 +145,9 @@ The first thing you should do is to make sure that the worker servers have [uniq
 
 Set up your worker processes on all worker servers (not the cluster nodes!) as windows services, but instead of using `NServiceBus.MSMQDistributor`, use `NServiceBus.MSMQWorker` profile instead.
 
+NOTE: Please use profiles **NServiceBus.Distributor** and **NServiceBus.Worker** when using NServiceBus earlier then V4.3
```

Comment:
i would prefer two headings here. "version x and higher" and "version x and lower". and explain the profiles in each